Pokemon Box/output_Flat.json

```diff
@@ -0,0 +1,627 @@
+          "`Classic L-Analog`": "..\\#DefaultDevices\\Wii_Remote\\Flat\\Classic L.png",
+          "`Classic R-Analog`": "..\\#DefaultDevices\\Wii_Remote\\Flat\\Classic R.png",
+          "`Classic Right`": "..\\#DefaultDevices\\Wii_Remote\\Flat\\Pad E.png",
+          "`Classic Up`": "..\\#DefaultDevices\\Wii_Remote\\Flat\\Pad N.png",
+          "`Classic Down`": "..\\#DefaultDevices\\Wii_Remote\\Flat\\Pad S.png",
+          "`Classic Left`": "..\\#DefaultDevices\\Wii_Remote\\Flat\\Pad W.png",
+          "`Classic Left X-`": "..\\#DefaultDevices\\Wii_Remote\\Flat\\Classic Left X-.png",
+          "`Classic Left X+`": "..\\#DefaultDevices\\Wii_Remote\\Flat\\Classic Left X+.png",
+          "`Classic Left Y-`": "..\\#DefaultDevices\\Wii_Remote\\Flat\\Classic Left Y-.png",
+          "`Classic Left Y+`": "..\\#DefaultDevices\\Wii_Remote\\Flat\\Classic Left Y+.png",
+          "`Classic Right X-`": "..\\#DefaultDevices\\Wii_Remote\\Flat\\Classic Right X-.png",
+          "`Classic Right X+`": "..\\#DefaultDevices\\Wii_Remote\\Flat\\Classic Right X+.png",
+          "`Classic Right Y-`": "..\\#DefaultDevices\\Wii_Remote\\Flat\\Classic Right Y-.png",
+          "`Classic Right Y+`": "..\\#DefaultDevices\\Wii_Remote\\Flat\\Classic Right Y+.png"
+        }
+      },
+  "generated_folder_name": "#Generated_Pokemon Box",
+  "preserve_aspect_ratio": true,
+  "output_textures": {
+	  "tex1_240x48_9c9e2800ccff0fbc_2.png": {
+      "image": "tex1_240x48_9c9e2800ccff0fbc_2.png",
+      "emulated_controls": {
+        "GCPad1": {
+          "Buttons/Start": [
+            [
+              488.0,
+              60.0,
+              920.0,
+              132.0
+            ]
+          ]
+        }
+      }
+    },
+    "tex1_32x32_6f854878154f4131_2.png": {
+      "image": "tex1_32x32_6f854878154f4131_2.png",
+      "emulated_controls": {
+        "GCPad1": {
+          "Buttons/Y": [
+            [
+              16.0,
+              12.0,
+              112.0,
+              112.0
+            ]
+          ]
+        }
+      }
+    },
+    "tex1_32x32_86a3e273b8413f2a_2.png": {
+      "image": "tex1_32x32_86a3e273b8413f2a_2.png",
+      "emulated_controls": {
+        "GCPad1": {
+          "Buttons/Y": [
+            [
+              0.0,
+              0.0,
+              128.0,
+              128.0
+            ]
+          ]
+        }
+      }
+    },
+    "tex1_32x32_bb1fd06002d87965_2.png": {
+      "image": "tex1_32x32_bb1fd06002d87965_2.png",
+      "emulated_controls": {
+        "GCPad1": {
+          "Buttons/X": [
+            [
+              0.0,
+              0.0,
+              128.0,
+              128.0
+            ]
+          ]
+        }
+      }
+    },
+    "tex1_32x32_d220acf64e3482f1_2.png": {
+      "image": "tex1_32x32_d220acf64e3482f1_2.png",
+      "emulated_controls": {
+        "GCPad1": {
+          "Buttons/X": [
+            [
+              16.0,
+              12.0,
```

Comment:
very nitpicky but,
i assume that they Buttons/X Y are used as animation?
12.0 =>16.0
so that it is centrit.
anyway good work 👍

Reply:
From what I could find, those buttons are used in completely seperate locations in the game, the larger ones are for the actual storage program, while the smaller ones are used in the "showcase" mode. However, the difference shouldn't be too noticable.